### ISSUE TITLE
请问八张A100训练时参数该如何设置？

### ISSUE THREAD

**Issue by 411104983** [deepinsight/insightface#2252]
您关于VIT-L的训练配置是使用64张卡，
我尝试了在webface42M上增大batchsize到1536,gradient_acc为2，目的是保证每个global-batch和您的保持一致。其他参数都保持不变，但观察到损失的下降并不稳定，同时epoch之间损失波动较大（我观察到您的epoch之间损失较为稳定），我是否因该降低我的学习率？
我该如何配置参数如学习率，batchsize以及梯度累加等，才可以在我的8张A100上尽可能的接近64张卡的效果？
感谢！

**Comment by Suvi-dha** [comment]
Hi @411104983 Did you find the answer to this? I am training VIT-L on 4 A6000 GPUs with 256 batch size on each GPU. Gradient accumulation is set at default value of 1. The learning rate is also 0.001. With these settings, after few epochs the loss stabilizes at 8 and is not further converging. Please help me if you could find the solution to this. Any help is much appreciated, Thanks!
